### DISCUSSION TITLE
Imports with ssr: false

### DISCUSSION THREAD

**Discussion by TheDarkStrix** [vercel/next.js#24459] [Help] [answered]
Hello, 

I want to import ReactQuill ,which gives me ReactQuill and Quill .
`import ReactQuill, { Quill } from "react-quill";`

How would import the same using ssr disabled .

As per the docs : 
```
const ReactQuill = dynamic(() => import("react-quill"), {
  ssr: false,
});
```
This will give me an instance of ReactQuill , but how do i also import `{Quill}` from this ?

Any help is very must appreciated .

**Comment by apollonian** [comment, marked as answer]
You could use a dynamic import for the named export from react-quill, like so:

```jsx
const DynamicQuill = dynamic(() =>
  import('react-quill').then((module) => module.Quill)
, { ssr: false })
```

https://nextjs.org/docs/advanced-features/dynamic-import#with-named-exports

**Reply by TheDarkStrix** [reply]
thanks !

**Reply by apollonian** [reply]
Happy to help!
